plugin/src/main/java/org/opensearch/ml/model/MLModelManager.java

```diff
@@ -787,6 +788,10 @@ public FunctionName getModelFunctionName(String modelId) {
         return modelCacheHelper.getFunctionName(modelId);
     }
 
+    public Optional<FunctionName> getOptionalModelFunctionName(String modelId) {
```

Comment:
Why we need this wrap function, not directly use modelCacheHelper.getOptionalFunctionName()?

Reply:
Model manager provides more functions which will make future development easier.

Reply:
Make sense, thanks.